browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -405,6 +408,13 @@ gfx::Size VerticalTabStripRegionView::GetMinimumSize() const {
 }
 
 void VerticalTabStripRegionView::Layout() {
+  if (!layout_dirty_ && last_size_ == size()) {
```

Comment:
Curious about this `last_size_ == size()`. In this PR, `last_size_` seems always empty.

Reply:
Uum,  doesn't line 416 fill that?

Reply:
Oops! I thought it's `Size()` :)